server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -161,14 +161,16 @@ const orderMutations = {
         throw new Error('Order must be active to be updated');
       }
 
+      const decodedPMId = getDecodedId(args.paymentMethod.id);
```

Comment:
Could be interesting to abstract that, similar to what we do with account `fetchAccountWithReference`. Ie: `fetchPaymentWithReference`.

Reply:
Happy to work on that perhaps in a separate PR starting tomorrow?